app/models/time_entry.rb

```diff
@@ -26,8 +26,8 @@ def self.to_csv
   end
 
   def self.filter_by_date(date)
-    entries = self.where("start_time >= ?", date.to_time)
-    entries.where("start_time < ?", (date + 1.day).to_time)
+    where('(start_time >= ? AND start_time < ?) OR running=TRUE',
```

Comment:
I think it's misleading to have a scope named `filter_by_date` that also includes entries that are still running. I believe a better approach would be to have a separate scope called `running`, and attempt to `merge` the two—or perhaps use Rails 5's `or`.